gulpfile.js

```diff
@@ -0,0 +1,12 @@
+const gulp = require('gulp');
```

Comment:
You don't need gulp for this project. You should try to use SASS through the builtin in create react app way.